### DISCUSSION TITLE
Zotero Obsidian workflow

### DISCUSSION THREAD

**Discussion by spandankeshav** [BryanJenksCommunity/FAQ#171] [Q&A] [answered]
​@BryanJenks Hi Bryan, I just recently implemented your zotero workflow in my system and works well. Thank you for this incredible piece of information. However, few things are not working as they should be.
1. In the keywords: It shows the tags with # instead of links [ ]?
2. Related section is empty even-though I have updated according to your instructions in the md file in zotero? 
3. I am seeing # sign in my title of the file e.g.   & 2023-05-12 # Title of the paper? Any idea why the # sign is there?
4. In my aliases section I see this "aliases: * Cite key: aliases. Do you know why the cite key is still embedded in the file?
5. My created and updated fields are empty when I export file from zotero even-though I have installed the "Update time on edit plugin"...
Is there anything I could do to make it work like you have showed? Thanks!

**Comment by tallguyjenks** [comment, marked as answer]
i just filmed a video on the zotero integration plugin workflow that i am now going to be using moving forward it should clear up all of these questions. Give it a week or two and it will be out ✅️

**Reply by spandankeshav** [reply]
Hi Bryan, great 👍. Looking forward to it. Cheese!
